configs/catalog.json

```diff
@@ -242,6 +242,54 @@
         },
         {
           "permission": "catalog:tenants:read"
+        },
+        {
+          "permission": "catalog:order_processes:create",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:order_processes:delete",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:order_processes:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:order_processes:write",
```

Comment:
@mkanoor Should this be `write` or `update`?

I'm concerned I see both uses which is inconsistent and confusing.

Reply:
I think this is probably supposed to be `update`, since we're doing `access.scopes(table_name, verb)`, and we will likely be passing in `"update"` as the verb.

Reply:
I followed the pattern for `order_item` and `order`, thought it will be same for `order_process`. In approval, there is no `write` action.

Reply:
Changed to `update`. Then in which situation we should use `write`?

Reply:
@hsong-rh Yeah, it's kinda weird. We don't currently check permissions against orders or order items. If you want to submit an order, the check that actually happens is "catalog:portfolio:order". We don't have any sort of permission check surrounding creating/updating/deleteing/listing orders or order items right now, so I think the "write" was just put there originally and since it was never used, it was never technically wrong.

Reply:
@mkanoor Please review and provide feedback.  I sense we want to be `using` update as part of the CRUD permissions we expose on the objects, unless `write` has a special meaning here.

Reply:
@gmcculloug update is the new verb previously we used to have write but that didn't allows us to distinguish between create and update. So write is no longer used. Currently we have write for things that the system creates like progress messages, order items we could convert them to 2 separate items with create/update down the road. Any objects that is controlled by a logged in user is using CRUD.

Reply:
@mkanoor Not for this PR, but would you recommend splitting the existing "write" permission into "create" and "update" permissions?

Reply:
@gmcculloug Yes we should try and split the write to create/update for objects that get exposed. Since this OrderProcess object is going to get exposed it makes sense to split the write as create and update.